elixir/lib/homework_web/resolvers/transactions_resolver.ex

```diff
@@ -60,10 +82,54 @@ defmodule HomeworkWeb.Resolvers.TransactionsResolver do
 
     case Transactions.delete_transaction(transaction) do
       {:ok, transaction} ->
-        {:ok, transaction}
+        {:ok, transaction |> to_dollars}
 
       error ->
         {:error, "could not update transaction: #{inspect(error)}"}
     end
   end
+
+  @doc """
+  For a transaction's amount field, converts the integer cents to it's decimal dollar equivalent
+  """
+  def to_dollars(%{amount: cents} = transaction) do
+    %{transaction | amount: cents |> to_dollars}
```

Comment:
I don't actually love piping everything on one line like this. At least as an Elixir newbie this made things harder to follow. But I deliberately left some of the code this way to show I followed it :P